tools/docker/nginx/vitamui-nginx.sh

```
@@ -0,0 +1,371 @@
+  compose up -d
+  wait_healthy
+  info ""
+  cmd_status
+}
+
+# nginx met une fraction de seconde a ouvrir son socket apres le `up`. Sans cette
+# attente, le `status` qui suit signale la sonde injoignable alors que tout va bien.
+wait_healthy() {
+  local origin i
+  origin="$(env_value VITAMUI_PUBLIC_ORIGIN https://dev.vitamui.com)"
+  for i in $(seq 1 20); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)